src/lib/lessons.ts

```diff
@@ -1,116 +1,149 @@
 import {LessonResource} from 'types'
 import {getGraphQLClient} from '../utils/configured-graphql-client'
 import getAccessTokenFromCookie from '../utils/get-access-token-from-cookie'
+import {loadLessonComments} from './lesson-comments'
+import {sanityClient} from 'utils/sanity-client'
+import groq from 'groq'
 
-export async function loadLesson(slug: string, token?: string) {
-  const query = /* GraphQL */ `
-    query getLesson($slug: String!) {
-      lesson(slug: $slug) {
-        slug
-        title
-        description
-        duration
-        next_up_url
-        free_forever
-        path
-        transcript
-        transcript_url
-        subtitles_url
-        hls_url
-        dash_url
-        http_url
-        media_url
-        lesson_view_url
-        thumb_url
-        icon_url
-        download_url
-        staff_notes_url
-        repo_url
-        code_url
-        created_at
-        updated_at
-        published_at
-        collection {
-          ... on Playlist {
-            title
+const lessonQuery = groq`
+*[_type == 'lesson' && slug == $slug][0]{
+  title,
+  slug,
+  description
+}`
```

Comment:
absolute base groq query for loading lesson metadata from Sanity, more to come here in followup PRs once we expand the Lesson schema.